### DISCUSSION TITLE
Is there a way to select the Portaudio backend?

### DISCUSSION THREAD

**Discussion by berarma** [performous/performous#947] [Q&A] [answered]
It uses ALSA by default and there are some issues with it. Sometimes it can't open the devices. The device list doesn't include some virtual sinks. I think this could be fixed by using the JACK backend.

Thanks.

**Comment by Baklap4** [comment]
@Lord-Kamina you know more about this than i do. Can you please answer it :)?

Also @berarma we do have a discord server which might be easier to ask questions in. Feel free to join https://discord.gg/NS3m3ad

**Reply by Lord-Kamina** [reply, marked as answer]
Yes you should be able to change it in the settings. In practice I've never tried it other than windows but it should work. 
Settings -> Audio -> Audio Backend

**Reply by Baklap4** [reply]
Thanks for the quick response: Settings -> Audio -> Audio Backend

Per default it uses Auto which is ALSA in your case i think

**Reply by Lord-Kamina** [reply]
Yeah, Auto is ALSA for most flavors of Linux, I believe. It's entirely arbitrary based on what I found by researching online at the time I coded the option to select, and thus could probably be refined if necessary.

**Comment by berarma** [comment]
Thanks! I don't know why I didn't find it before.
